code_to_gast/helpers/py_helpers.py

```diff
@@ -151,4 +151,10 @@ def name_to_gast(node):
 takes node of type unaryOp and converts it to our generic ast represenations
 """
 def unary_op_to_gast(node):
-    return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
\ No newline at end of file
+    return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
+
+"""
+takes argument from function declaration and makes into gast node
+"""
+def arg_to_gast(node):
+    return {"type": "name", "value": node.arg}
```

Comment:
Does this handle complex values? seems like value should maybe come from another call to `node_to_gast`.

Reply:
+1